models/Device.js

```diff
@@ -32,10 +30,26 @@ module.exports = function(sequelize, DataTypes) {
     },
   };
 
+  var allForUser = async function(user) {
+    var deviceIds = await user.getDeviceIds();
+    var userGroupIds = await user.getGroupsIds();
+    console.log("deviceIds", deviceIds);
+    console.log("userGroupIds", userGroupIds);
```

Comment:
We don't want to use `console.log()` do we?

Reply:
No we don't. That's from when I was checking something.